security_monkey/0.7.0/config-deploy.py

```diff
@@ -0,0 +1,305 @@
+    sm_config['postgres']['host'] = os.environ.get('SECURITY_MONKEY_POSTGRES_HOST')
+
+if 'SECURITY_MONKEY_POSTGRES_USER' in os.environ:
+    sm_config['postgres']['user'] = os.environ.get('SECURITY_MONKEY_POSTGRES_USER')
+
+if 'SECURITY_MONKEY_POSTGRES_PASSWORD' in os.environ:
+    sm_config['postgres']['password'] = os.environ.get('SECURITY_MONKEY_POSTGRES_PASSWORD')
+
+if 'SECURITY_MONKEY_POSTGRES_DATABASE' in os.environ:
+    sm_config['postgres']['database'] = os.environ.get('SECURITY_MONKEY_POSTGRES_DATABASE')
+
+if 'SECURITY_MONKEY_POSTGRES_PORT' in os.environ:
+    sm_config['postgres']['port'] = os.environ.get('SECURITY_MONKEY_POSTGRES_PORT')
+
+if 'SECURITY_MONKEY_API_PORT' in os.environ:
+    sm_config['api']['port'] = os.environ.get('SECURITY_MONKEY_API_PORT')
+
+if 'SECURITY_MONKEY_FQDN' in os.environ:
+    sm_config['fqdn'] = os.environ.get('SECURITY_MONKEY_FQDN')
+
+if 'SECURITY_MONKEY_SECURITY_TEAM_EMAIL' in os.environ:
+    sm_config['email']['security-team-email'] = os.environ.get('SECURITY_MONKEY_SECURITY_TEAM_EMAIL')
+
+if 'SECURITY_MONKEY_SMTP' in os.environ:
+    # Must change String from environment variable into Boolean
+    if os.environ.get('SECURITY_MONKEY_SMTP') == 'True':
+        sm_config['email']['smtp'] = True
+
+if 'SECURITY_MONKEY_SES_REGION' in os.environ:
+    sm_config['email']['ses-region'] = os.environ.get('SECURITY_MONKEY_SES_REGION')
+
+if 'SECURITY_MONKEY_EMAIL_DEFAULT_SENDER' in os.environ:
+    sm_config['email']['default-sender'] = os.environ.get('SECURITY_MONKEY_EMAIL_DEFAULT_SENDER')
+
+if 'SECURITY_MONKEY_EMAIL_SERVER' in os.environ:
+    sm_config['email']['server'] = os.environ.get('SECURITY_MONKEY_EMAIL_SERVER')
+
+if 'SECURITY_MONKEY_EMAIL_USERNAME' in os.environ:
+    sm_config['email']['username'] = os.environ.get('SECURITY_MONKEY_EMAIL_USERNAME')
+
+if 'SECURITY_MONKEY_EMAIL_PASSWORD' in os.environ:
+    sm_config['email']['password'] = os.environ.get('SECURITY_MONKEY_EMAIL_PASSWORD')
+
+LOG_CFG = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'standard': {
+            'format': '%(asctime)s %(levelname)s: %(message)s '
+                '[in %(pathname)s:%(lineno)d]'
+        }
+    },
+    'handlers': {
+        'file': {
+            'class': 'logging.handlers.RotatingFileHandler',
+            'level': 'DEBUG',
+            'formatter': 'standard',
+            'filename': '/var/log/security_monkey/securitymonkey.log',
+            'maxBytes': 10485760,
+            'backupCount': 100,
+            'encoding': 'utf8'
+        },
+        'console': {
+            'class': 'logging.StreamHandler',
+            'level': 'DEBUG',
+            'formatter': 'standard',
+            'stream': 'ext://sys.stdout'
+        }
+    },
+    'loggers': {
+        'security_monkey': {
+            'handlers': ['file', 'console'],
+            'level': 'DEBUG'
+        },
+        'apscheduler': {
+            'handlers': ['file', 'console'],
```

Comment:
Probably don't want to use `file` in docker.

Logging should go to stdout, iirc.